docs/specs/servers/blockchain/blockchain-server-api.md

```diff
@@ -276,3 +276,127 @@ The POST request body should be in JSON format with the following structure:
 
 * `400 Bad Request` - some parameters in request body were missed or wrong.
 * `401 Unauthorized` - projectID verification error.
+
+## Conversion
+
+Endpoints to make convert token transactions.
+
+### List of tokens available for conversion
+
+`GET /v1/convert/tokens`
+
+#### Path parameters
+
+* `projectId` - is the project identifier
+* `chainId` - filter by chain. CAIP-2 format. e.g. `eip155:1`
+
+#### Success response body:
+
+* `tokens` - list of objects which define available tokens:
+    * `name` - token name.
+    * `symbol` - token symbol.
+    * `address` - contract address of the token in CAIP-10 format.
```

Comment:
Really using CAIP-10 to identify a contract? Never seen that before but I guess it works 🤔

Reply:
We want the responses/requests to be chain agnostic as much as possible, so probably this is a good way.